cli/inspector_structures.rs

```diff
@@ -0,0 +1,425 @@
+use deno_core::serde_json;
+use deno_core::serde_json::Value;
+use serde::Deserialize;
+use serde::Serialize;
+
+#[derive(Debug, Clone, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AwaitPromiseArgs {
```

Comment:
Could you add links to relevant message in https://chromedevtools.github.io/devtools-protocol/tot/ as docstrings?